pootle/templates/editor/_actions.html

```diff
@@ -2,7 +2,7 @@
 <ul>
   <li class="back-to-browser">
     <a class="js-back-to-browser" href="{{ previous_url }}">
-      <span class="caption">{% trans 'Go back to overview' %}</span>
+      <span class="caption">{% trans 'Go back to browser' %}</span>
```

Comment:
Any of those two (*Go back to browsing* or *Go back to browse view*) is fine for me.